credsweeper/app.py

```diff
@@ -76,10 +73,15 @@ def __init__(self,
         self.json_filename: Optional[str] = json_filename
         self.ml_batch_size = ml_batch_size
         self.ml_threshold = ml_threshold
-        if ml_validation:
+        if self._use_ml_validation():
```

Comment:
As soon we make ML as main feature - import should be used as usually. And setup.py ...

Reply:
Yes, I'll handle that feature after this PR. We should change `setup.py`, but it is not that easy feature.
Actually now we support `pip install credsweepr[ml]` option for ml compatibility. So we should discuss about it. Remove `[ml]` installation option or some other way.

Reply:
OK.
IMHO CredSweeper is used due ML

Reply:
> OK. IMHO CredSweeper is used due ML

+